### DISCUSSION TITLE
Leveraging GoAkt for In-Memory Data Grid Implementation

### DISCUSSION THREAD

**Discussion by civilizeddev** [Tochemey/goakt#138] [Ideas]
I have an idea to use [goakt](https://github.com/Tochemey/goakt) as a foundation for implementing an in-memory data grid. How about adding it to the [examples](https://github.com/Tochemey/goakt/tree/main/examples)? There's a blog post about it. https://hazelcast.com/glossary/in-memory-data-grid

**Comment by Tochemey** [comment]
@civilizeddev feel free to add it. You can always reach out for help. I am also open to any suggestions to make the framework better.

**Reply by Tochemey** [reply]
@civilizeddev maybe this project may inspire you https://github.com/Tochemey/ego. I can't really tell. I will take a look at the data-grid post.

**Comment by Tochemey** [comment]
@civilizeddev, may I ask what's next with this idea?

**Reply by civilizeddev** [reply]
Sorry, right now it's just a distant idea of a grand dream. For me, getting familiar with goakt comes first.

I happen to have a problem to solve, so I'm thinking of using goakt. I've had bad experiences with [asynkron/protoactor-go](https://github.com/asynkron/protoactor-go) before, so I've been hesitant to use actor-based solutions, but I'll see if goakt can be a substitute.

**Reply by Tochemey** [reply]
@civilizeddev I can be of help whenever you need. Just dm me. Go-Akt comes from my experience of using Akka and Proto-Actor.

**Comment by Tochemey** [comment]
@civilizeddev closing this discussion for lack of activities
